README.md

```diff
@@ -15,8 +15,219 @@ macOS...sorry :-)
 
 `hugo new doc/add-this-doc.md`
 
-### Making changes to CSS
+If you'd like to preview your doc updates as you work, you can install Hugo
```

Comment:
This is only a part of the updates to the README file on the master branch. We should include all the updates, as this bit looks odd when placed immediately after the non-updated part.